circus-train-api/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>circus-train-parent</artifactId>
-    <version>16.2.1-SNAPSHOT</version>
+    <version>16.1.1-SNAPSHOT</version>
```

Comment:
Shouldn't this be 16.2.0-SNAPSHOT? (although you can override it in Jenkins when you do the release so doesn't matter too much)

Reply:
I just reverted the changes that the maven release made and put it back to what it was before, so when I release I'll add the version then.